lib/web_console/middleware.rb

```diff
@@ -79,12 +81,20 @@ def create_regular_or_whiny_request(env)
         whiny_requests ? WhinyRequest.new(request) : request
       end
 
+      def mount_point
```

Comment:
You actually get an instance `mount-point` method from `cattr_accessor`. If you do the normalization in the Railtie, during initialization, you can drop this method.